src/utils.js

```diff
@@ -14,21 +14,26 @@ export { default as union } from 'lodash/union'
 
 export { onLoad } from '@launchpadlab/lp-utils'
 
+// Returns true when two arrays share common items
 export function hasOverlap (arr1, arr2) {
   const overlapItems = overlap(arr1, arr2)
   return (overlapItems.length > 0)
 }
 
+// Converts all keys of the given object to camelCase
 export function camelizeKeys (obj) {
```

Comment:
the docs should mention that this is wrapper around `humps` and the docs and test should address the exceptional case when a key starts with a `_`, it is not transformed.